packages/react-ui/__tests__/widgets/FormulaWidgetUI.test.js

```diff
@@ -3,15 +3,6 @@ import { render, screen } from '@testing-library/react';
 import FormulaWidgetUI from '../../src/widgets/FormulaWidgetUI';
 import { currencyFormatter } from './testUtils';
 
-// Mock animations
```

Comment:
I added this cause I noticed these tests were unstable, when using the real animation steps... I guess that adding the rerender + await removes the problem, but just for you to know

Reply:
Yep, [check this out](https://testing-library.com/docs/guide-disappearance/#1-using-findby-queries)